addons/html_builder/static/src/snippets/snippet_service.js

```diff
@@ -104,7 +102,9 @@ export class SnippetModel extends Reactive {
                     // });
                 } catch (e) {
                     if (e instanceof RPCError) {
-                        const message = escape(_t("Could not install module %s", snippet.title));
```

Comment:
Is the old editor impacted as well? It feels like something that should be done as a fix to avoid double escaping (and therefore incorrect rendering) if the translated text or the title contains escapable chars.

Reply:
this code is only run in the new editor. i assume double escaping is not a big deal since snippet.title is actually the name of a module, which should be reasonably simple.

Reply:
But the same mistake was present in `web_editor` too, it seems the line was just copied from there, so the issue must exist in all versions.

If the French translation is `Impossible d'installer le module...` it will display as `Impossible d&#x27;nstaller le module...`. It's not only the module name the issue but also the text itself. 

Just testing with quote in the text to confirm:
![image](https://github.com/user-attachments/assets/39c9ea86-5955-47fb-beb8-418fa979c38c)